myuw/data/quicklinks.csv

```diff
@@ -1,11 +1,14 @@
 URL,Label,campus,affiliation,pce
-http://www.washington.edu/maps/,Campus Maps,,,
-http://canvas.uw.edu/,Canvas LMS,,employee,
+http://canvas.uw.edu/,Canvas LMS,,faculty,
 http://canvas.uw.edu/,Canvas LMS,,student,
-https://catalyst.uw.edu/,Catalyst Web Tools,,,
-https://sdb.admin.uw.edu/students/uwnetid/address.asp,Change Student Address,,,
-https://www.washington.edu/ess/,WorkDay,,,
-http://www.washington.edu/home/peopledir/,Faculty/Staff/Student Directory,,,
-https://itconnect.uw.edu/,IT Connect,,,
-https://panopto.uw.edu/,Panopto,,,
-https://sdb.admin.uw.edu/students/uwnetid/register.asp,Registration,,,
+https://myplan.uw.edu,MyPlan,,student,
+https://sdb.admin.uw.edu/sisStudents/uwnetid/grades.aspx,Grade Report,,student,
+https://www.washington.edu/ess/,WorkDay,,employee,
```

Comment:
Should ESS be called "WorkDay"? This will go out June 6, so  wonder if this should be "Employee Self-Service (ESS)"

Once Workday is out, we could maybe add it as another QL?